test/e2e/storage/vsphere/vsphere_volume_vsan_policy.go

```diff
@@ -320,7 +321,7 @@ func invokeInvalidPolicyTestNeg(client clientset.Interface, namespace string, sc
 	return fmt.Errorf("Failure message: %+q", eventList.Items[0].Message)
 }
 
-func invokeStaleDummyVMTestWithStoragePolicy(client clientset.Interface, namespace string, clusterName string, scParameters map[string]string) {
+func invokeStaleDummyVMTestWithStoragePolicy(client clientset.Interface, nodeInfo *NodeInfo, namespace string, clusterName string, scParameters map[string]string) {
```

Comment:
Why not pass node name here as well?

Reply:
passed master node name instead of `nodeinfo`